ghostTemplate.yaml

```diff
@@ -0,0 +1,377 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  ghostFS:
+    Type: AWS::EFS::FileSystem
+    Properties:
+      PerformanceMode: generalPurpose
+      Encrypted: false
+  ghostMount0:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 0 
+        - !Ref DefaultVPCSubnets
+  ghostMount1:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 1 
+        - !Ref DefaultVPCSubnets
+  ghostMount2:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 2 
+        - !Ref DefaultVPCSubnets
+  ghostMount3:
+    Type: AWS::EFS::MountTarget
+    Properties: 
+      FileSystemId: !GetAtt ghostFS.FileSystemId
+      SecurityGroups: 
+        - !GetAtt ghostEFSSecurityGroup.GroupId
+      SubnetId: !Select 
+        - 3 
+        - !Ref DefaultVPCSubnets
```

Comment:
I think we only want to create ghostMount0 and ghostMount1. There is one default subnet for each AZ in the region this is deployed to, but there are only guaranteed to be two AZs in any region. This is an article showing how many AZs there were in each region at its publication time demonstrating this: https://medium.com/@jaychapel/5-things-you-need-to-know-about-aws-regions-and-availability-zones-60af79b718f9.

Reply:
Interesting. Thanks for the post! 

![Screen Shot 2020-11-01 at 4 09 15 PM](https://user-images.githubusercontent.com/1148878/97820149-9e68b600-1c61-11eb-8796-d1efd7b01701.png)

I've been testing this implementation out and while functional, the ECS service will take longer to become healthy because of task invocations that sit on a subnet without a mount target. I suppose that's fine for this stack, though I should note it in our docs.

Reply:
I'm leaving comments in a follow up review of the template to address this.